src/utils/samlUtils.ts

```diff
@@ -77,6 +79,99 @@ const SAMLResponse = t.type({
   SAMLResponse: t.string
 });
 
+export const InfoNotAvailable = "NOT AVAILABLE";
+
+/**
+ * If an eventHandler and a feature flag are provided this function logs the timing deltas.
+ * This is useful to monitor the timings and to adjust the clockSkewMs variable
+ */
+export const extractAndLogTimings = (
```

Comment:
All the time check on this function are not considering the skew. This means that an alert based on this metrics should be changed any time we modify the skew.

Reply:
Do you think it would be better to remove the skew from the deltas and have an "outbound timing difference" just to indicate by how much we passed the skew? Personally I prefer a complete difference logging, we can compare deltas with the skew and set up an alert accordingly.

Reply:
let's discuss more on this before merging.

Reply:
added clockSkew variable in the calculations. changes are in https://github.com/pagopa/io-spid-commons/pull/121/commits/ef332286fa24914327e43056fae8b9ac0b234230